src/frequenz/sdk/timeseries/_moving_window.py

```diff
@@ -318,6 +320,44 @@ def window(
             start, end, force_copy=force_copy, fill_value=fill_value
         )
 
+    async def wait_for_samples(self, n: int) -> None:
```

Comment:
Maybe:
```suggestion
    async def wait_for_samples(self, n: int, /) -> None:
```
as using it as a keyword doesn't really bring anything.

Reply:
Not doing it, because what it someone wants to.